### PR TITLE
M3C: overclocked_electromancer + support

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-game/src/main/java/forge/game/card/Card.java [hunk 1]
```diff
@@ -269,6 +269,7 @@ public class Card extends GameEntity implements Comparable<Card>, IHasSVars {
     private Map<Integer, Integer> damage = Maps.newHashMap();
     private boolean hasBeenDealtDeathtouchDamage;
     private boolean hasBeenDealtExcessDamageThisTurn;
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/card/Card.java:271]: Can this variable be removed if excessDamageThisTurnAmount > 0?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/card/Card.java:271]: I wondered about that... I think so. I was just worried about the LKI part in CardDamageMap line 131

Comment (tool4ever) [reply forge-game/src/main/java/forge/game/card/Card.java:271]: yea the logic for _Rith, Liberated Primeval_ probably needs to be refactored for that to work:
> if a creature or planeswalker an opponent controlled was dealt excess damage this turn

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/card/Card.java:271]: Maybe leave both for now and see what WotC does next? They seem to be experimenting with "excess damage" right now.

**Review by tehdiplomat — APPROVED** [review]

**Pull request merged**